packages/geolocation/src/drivers/mongodb/populate.js

```diff
@@ -6,12 +6,10 @@ const Logger = require('../../utils/logger');
 const { placesSchema, populate } = require('../../utils/syncDatabase');
 
```

Comment:
Remeber to change the way we import logger

a) if you need to use Logger const { Logger } = require('@booking-services/shared');
b) if you need to reference logger file @booking-services/shared/src/utils/logger

Reply:
Can you check it, did I make the changes?

Reply:
Yeah that part is okay now